### PR TITLE
Reorganized last class (python 210) stuff into folders.

### PR DESCRIPTION
**Pull request by HarryMaher**

### CODE REVIEW

**Comment by HarryMaher** [comment]
Srry, I should have done separate commits for cleaning up my folder and putting everything into a new directory.
Here's a link to session01 homework:
https://github.com/HarryMaher/Wi2018-Classroom/tree/master/students/Harry_Maher/Python220B/session01

File: students/Harry_Maher/Python220B/session01/iterator_1.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    for i in IterateMe_2(2, 20, 2):
+        print(i)
+    print("break test")
+    it = IterateMe_2(2, 20, 2)
+    for i in it:
+        if i > 10:  break
+        print(i)
+    print("break, then starts up again")
+    for i in it:
+        print(i)
```
Comment (davidblaszka) [inline students/Harry_Maher/Python220B/session01/iterator_1.py]: This looks pretty good, but can you make it work like range(). In other words, we want to run it through a loop. Stop the loop. Then rerun it from the beginning.

File: students/Harry_Maher/Python220B/session01/generator_solution.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    """Someone wrote a test for a second one of these? Y?"""
+    tot = 0
+    n = 1
+    while "arrrrrrr":
+        yield tot
+        tot = sum([tot,n])
+        n += int(True)
+
+
+def doubler(n=1):
```
Comment (davidblaszka) [inline students/Harry_Maher/Python220B/session01/generator_solution.py]: short and concise. Nice work!

File: students/Harry_Maher/Python220B/session01/topmusic2017.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+# In[4]:
+
+
+music.describe()
+
+
+# In[11]:
+
+
+songs = [x for x in list(zip(music.danceability, music.loudness, music.artists, music.name)) if x[0] > .8 and x[1] < -5.0]
```
Comment (davidblaszka) [inline students/Harry_Maher/Python220B/session01/topmusic2017.py]: I like how you harnessed all of this inside a list comprehension.

File: students/Harry_Maher/Python220B/session01/topmusic2017.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+# In[12]:
+
+
+songs
+
+
+# In[17]:
+
+
+songs = sorted(songs) # Yeah I should have put them in a different order and used a key, but lazy.
```
Comment (davidblaszka) [inline students/Harry_Maher/Python220B/session01/topmusic2017.py]: check out help(sorted)

File: students/Harry_Maher/Python220B/session01/topmusic2017.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+
+# In[18]:
+
+
+top_5_songs = songs[::-1][:5]
```
Comment (davidblaszka) [inline students/Harry_Maher/Python220B/session01/topmusic2017.py]: hmmm, looks like you could try sorted

**Pull request merged**